sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/InlineCTE.scala

```diff
@@ -225,12 +233,15 @@ case class InlineCTE(
  *                 from other CTE relations and regular places.
  * @param outgoingRefs A mutable map that tracks outgoing reference counts to other CTE relations.
  * @param shouldInline If true, this CTE relation should be inlined in the places that reference it.
+ * @param container The container of a CTE definition is another CTE definition in witch the
```

Comment:
```suggestion
 * @param container The container of a CTE definition is another CTE definition in which the
```

Reply:
Done in https://github.com/apache/spark/pull/49379/commits/c62c6c4bf2ec4591fee1c5129315a2d9878159a9.